app/services/import_notifier.rb

```diff
@@ -56,7 +49,8 @@ def find_meters_running_behind
          .joins('LEFT JOIN data_sources on data_sources.id = meters.data_source_id')
          .joins(:amr_validated_readings)
          .where(schools: { active: true })
-         .group('meters.id, data_sources.import_warning_days')
+         .joins('LEFT JOIN school_groups on schools.id = school_groups.id')
```

Comment:
should this be `.joins('LEFT JOIN school_groups ON schools.school_group_id = school_groups.id')`?